CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 All notable changes to this project will be documented in this file.
 We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
+## 2015-04-08
```

Comment:
There's a note above about semver but none of these changes include release numbers. Should that note be removed?